port/port_posix_sse.cc

```diff
@@ -54,7 +54,7 @@ static inline bool HaveSSE42() {
   __cpuid(cpu_info, 1);
   return (cpu_info[2] & (1 << 20)) != 0;
 #elif defined(__GNUC__)
-  unsigned int eax, ebx, ecx, edx;
+  unsigned int eax, ebx, ecx = 0, edx;
   __get_cpuid(1, &eax, &ebx, &ecx, &edx);
   return (ecx & (1 << 20)) != 0;
```

Comment:
__get_cpuid can fail if the function parameter isn't supported by the hardware (which won't be the case for any supported CPU), in which case it does not modify eax/ebx/ecx/edx.